core/inst_handlers/f/RvfInsts.cpp

```diff
@@ -1,112 +1,137 @@
-#include "core/inst_handlers/rv64/f/RvfInsts.hpp"
+#include "core/inst_handlers/f/RvfInsts.hpp"
 #include "include/ActionTags.hpp"
 #include "core/ActionGroup.hpp"
 
 namespace atlas
 {
+    template <typename XLEN>
     void RvfInsts::getInstComputeAddressHandlers(std::map<std::string, Action> & inst_handlers)
     {
-        inst_handlers.emplace(
-            "flw", atlas::Action::createAction<&RvfInsts::flw_64_compute_address_handler, RvfInsts>(
-                       nullptr, "flw", ActionTags::COMPUTE_ADDR_TAG));
-        inst_handlers.emplace(
-            "fsw", atlas::Action::createAction<&RvfInsts::fsw_64_compute_address_handler, RvfInsts>(
-                       nullptr, "fsw", ActionTags::COMPUTE_ADDR_TAG));
-    }
-
+        static_assert(std::is_same_v<XLEN, RV64> || std::is_same<XLEN, RV32>::value);
```

Comment:
You might as well use std::is_same_v in all places.

Reply:
Bad `sed` command! I'll fix it. Good catch.